src/cdk/v2/destinations/bluecore/data/bluecoreIdentifyConfig.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "destKey": "properties.distinct_id",
```

Comment:
can this come up from externalId as well ?

Reply:
yes! re-read the doc once again, adding support